src/classes/DragNDropController.cls

```diff
@@ -0,0 +1,88 @@
+public with sharing class DragNDropController {
+
+/**
```

Comment:
The header class should be the first thing on the class. Please move it above.
Also add one blank line between description and @author, remove the parenthesis that wraps the date and please align the "*" as shown in the picture above.